libs/application/templates/estate/src/fields/RealEstateRepeater/AdditionalRealEstate.tsx

```diff
@@ -65,8 +65,21 @@ export const AdditionalRealEstate = ({
     // https://www.skra.is/um-okkur/frettir/frett/2018/03/01/Nytt-fasteignanumer-og-itarlegri-skraning-stadfanga/
     // The property number is a seven digit informationless sequence with prefix F
     // The lot number is a six digit informationless sequence with prefix L
-    const propertyNumber = propertyNumberInput.trim().toUpperCase()
+    let propertyNumber = propertyNumberInput.trim().toUpperCase()
     setValue(addressField, '')
+
+    // check if property number starts with a number
+    if (propertyNumber && !isNaN(propertyNumber.charAt(0))) {
+      const str = String(propertyNumber).trim()
+      if (str.length === 7) {
+        propertyNumber = `F${str}`
+      }
+
+      if (str.length === 6) {
+        propertyNumber = `L${str}`
+      }
+    }
```

Comment:
I'm wondering, would it be simpler to just allow lookup with pure numbers in front instead of requiring `L` or `F`?

Modifying this here to have Ll and fF as optional?

https://github.com/island-is/island.is/blob/c0f8107bdf785d81c3b0635bab4c5fd0e9746d83/libs/application/templates/inheritance-report/src/lib/utils/helpers.ts#L82